modules/api/php/provisioners/projectdicomsrowprovisioner.class.inc

```diff
@@ -0,0 +1,100 @@
+<?php declare(strict_types=1);
+/**
+ * This file implements a data provisioner to get all dicoms of a project
+ * created since a given date.
+ *
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\api\Provisioners;
+
+use \LORIS\Data\Provisioners\DBObjectProvisioner;
+use \LORIS\api\Models\ProjectDicomsRow;
+/**
+ * This file implements a data provisioner to get all dicoms of a project
+ * created since a given date.
+ *
+ * PHP Version 7
+ *
+ * @category API
+ * @package  Loris
+ * @author   Xavier Lecours Boucher <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class ProjectDicomsRowProvisioner extends DBObjectProvisioner
+{
+    /**
+     * Create a RowProvisioner
+     *
+     * @param \Project  $project   The project from which dicoms are requested
+     * @param \DateTime $since     The date from which dicoms are requested
+     * @param string    $classname The class name of the returned objects
+     */
+    function __construct(
+        \Project $project,
+        \DateTime $since,
+        string $classname='default'
```

Comment:
```suggestion
        string $classname='ProjectDicomsObjectProvisioner'
```

Reply:
That would be `\LORIS\api\Models\ProjectDicomsRow.class.inc`